openshift/oc_promote.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#%
+#% OpenShift Deployment Promotion Helper
+#%
+#%   Intended for use with a pull request-based pipeline.
+#%
+#% Usage:
+#%
+#%   ${THIS_FILE} [PR_NUMBER] [apply]
+#%
+#% Examples:
+#%
+#%   Provide a PR number. Defaults to a dry-run.
+#%   ${THIS_FILE} 0
+#%
+#%   Apply when satisfied.
+#%   ${THIS_FILE} 0 apply
+#%
+#
+# If no parameters, then show this help header (cat file, grep #% lines and clean up with sed)
+#
+[ "${#}" -gt 0 ] || {
```

Comment:
Can we do [ "${#}" -eq 0 ] && { ... } instead?
I feel this way is easier to read

Reply:
Cause with `&&` you are saying 'okay if this condition is met then do the following'

Reply:
@Kyubinhan if the && condition fails, so will the script.  Bash is rather paticular.